packages/core/src/components/non-ideal-state/_non-ideal-state.scss

```diff
@@ -12,8 +12,10 @@ Markup:
   <div class="#{$ns}-non-ideal-state-visual">
     <span class="#{$ns}-icon #{$ns}-icon-folder-open"></span>
   </div>
-  <h4 class="#{$ns}-heading">This folder is empty</h4>
-  <div>Create a new file to populate the folder.</div>
+  <div class="#{$ns}-non-ideal-state-text">
```

Comment:
This is another minor change to the generated DOM, but I confirmed that the styles are compatible with the old markup when both title and description are present:

<img width="1372" alt="image" src="https://user-images.githubusercontent.com/723999/166486172-dda268e9-6ddc-436b-897c-3307bbeb6fdc.png">

I'll have to make a small change to make it work when there's only a title... or we might just accept the small break to the CSS API in that case (we'll just see a 10px margin instead of a 20px one iff using the old markup).